### PR TITLE
Improve Quote/Cashe Docs

### PR DESCRIPTION
**Pull request by JasonAndrewWriter**
Rendered pages: https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/storage/application-tool?branch=pr-en-us-2176

https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/progressive-web-apps/?branch=pr-en-us-2176#quota-usage

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/storage/cache.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ ms.date: 05/04/2021
 
 This guide shows you how to use DevTools to inspect [Cache](https://developer.mozilla.org/docs/Web/API/Cache) data.
 
-To inspect [HTTP cache](https://developer.mozilla.org/docs/Web/HTTP/Caching) data, in the **Network** tool, use the **Size** column of the **Network Log**.  See [Log network activity](../network/index.md#log-network-activity) in _Inspect network activity_.
+To inspect [HTTP cache](https://developer.mozilla.org/docs/Web/HTTP/Caching) data, in the **Network** tool, use the **Size** column of the **Network Log**.  See [Log network activity](../network/index.md#log-network-activity) in _Inspect network activity_.  See [Quota Useage](https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/progressive-web-apps/#quota-usage)
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/storage/cache.md:27]: Same comments here, the link here seems a bit arbitrary. There should be a clearer explanation about why we're linking to this particular page, and I don't think it belongs in this paragraph.

Also, the link is incorrect: it shouldn't be absolute but should instead be a relative path. Also, when you need to use an absolute URL (when linking to a 3rd-party site for example), don't include the `/en-us` part. This is usually automatically added by the site depending on the preferred language of the user.

So, in this case, I would do the following: at the end of the "View cache data" section, add a paragraph with something like: "To learn more about cache data storage quota, see ..."

**Review by captainbrosset — CHANGES_REQUESTED** [review]

File: microsoft-edge/webview2/concepts/user-data-folder.md [hunk 2]
```diff
@@ -416,7 +416,7 @@ On this platform, use the default UDF location.
 
 **Example code:**
 
-For example code, see the WinUI 3 `.cs` file, at [WebView2Samples repo > WebView2_WinUI3_Sample](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/WebView2_WinUI3_Sample).
+For example code, see the WinUI 3 `.cs` file, at [WebView2Samples repo > WebView2_WinUI3_Sample](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/GettingStartedGuides/WinUI3_GettingStarted).
```
[the remark this answers is not in the document]
Comment (JasonAndrewWriter) [reply microsoft-edge/webview2/concepts/user-data-folder.md:419]: My bad,  I'll make sure to fix.

File: microsoft-edge/devtools-guide-chromium/storage/application-tool.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ The Application tool has the following pages:
    *  Interest Groups
 
 *  Cache
-   *  Cache Storage
+   *  [Cache Storage](https://learn.microsoft.com/microsoft-edge/devtools-guide-chromium/progressive-web-apps/#quota-usage)
```
[the remark this answers is not in the document]
Comment (JasonAndrewWriter) [reply microsoft-edge/devtools-guide-chromium/storage/application-tool.md]: fixed

File: microsoft-edge/devtools-guide-chromium/storage/cache.md (hunk 1 above)
Comment (JasonAndrewWriter) [reply microsoft-edge/devtools-guide-chromium/storage/cache.md]: fixed

**Review by JasonAndrewWriter — COMMENTED** [review]
I made your changes and removed user-data-folder.md.  Not sure how it ended up in here.

**Comment by JasonAndrewWriter** [comment]
closed this pr in favor of https://github.com/MicrosoftDocs/edge-developer/pull/2192

**Pull request closed without merging**